### PR TITLE
feat: workdir behavior changed and add target input

### PR DESCRIPTION
**Pull request by Milly**
## related issue

no

## Fix or Add this PR/このプルリクエストでやったこと

- `workdir` changes current directory. Now, this is not a target directory.
  - This is **breaking change**. Must be update version to v2.
- `target` is added. This is the target path of mypy command. 
- Added tests for above.
- *.mypy_cache/* is added to *.gitignore*. (for testing in local)

## Not fix or add this PR/このプルリクエストでやらなかったこと

no

## Pros and Cons/メリットとデメリット

Because *mypy.ini* is only used at the current directory.
If *mypy.ini* is exists in the sub-directory, can not detect errors.

## Test/動作確認

- [x] The tests fails, before modifying the code. (*jobs.test-local.steps.mypy-local-error-should-failure* should failed)
- [x] The tests success, after modifying the code. (All of *jobs.test-local* should succeeded)

### CODE REVIEW

**Comment by Milly** [comment]
適切な semantic PR title はなんでしょうね……

**Comment by tsuyoshicho** [comment]
LGTM! (とりあえず見た限り)

後で詳細に確認します。

> 適切な semantic PR title はなんでしょうね……

feat: かな? まあ、これは満せてなくてもマージ判断には加えませんからw (あくまでコミットログ他のためにやってるだけで、必須ではないつもりです)

**Review by tsuyoshicho — CHANGES_REQUESTED** [review]
Please Update README for this change.

- add sample and param
- not need v1 and v2 differences point/wrote "workdir" change current dir info

---

上記が追記できたらmajor verup としてマージします。

**Comment by Milly** [comment]
README and description in *action.yml* are changed.

File: README.md [hunk 1]
```diff
@@ -83,6 +91,9 @@ jobs:
           # Change reporter level if you need.
           # GitHub Status Check won't become failure with warning.
           level: warning
+          # Change the current directory to run mypy command.
+          # mypy commmand reads setup.cfg or other settings file in this path.
```
Comment (tsuyoshicho) [inline README.md:95]:
```suggestion
          # mypy command reads setup.cfg or other settings file in this path.
```

**Review by tsuyoshicho — APPROVED** [review]

**Comment by tsuyoshicho** [comment]
Thank you for your contribution!

File: README.md (hunk 1 above)
Comment (Milly) [reply README.md]: Oh. sorry

**Pull request merged**